CADETProcess/processModel/flowSheet.py

```diff
@@ -314,12 +365,31 @@ def connections(self):
         add_connection
         remove_connection
 
+        """
+        """
```

Comment:
[Question] Is this code still relevant?

Reply:
Passing the ball to @daklauss , was this something we tried and then dropped?